config/dev/openstack-clusterdeployment.yaml

```diff
@@ -4,9 +4,11 @@ metadata:
   name: openstack-dev
   namespace: ${NAMESPACE}
 spec:
-  template: openstack-standalone-cp-0-0-3
+  template: openstack-standalone-cp-0-0-5
   credential: openstack-cluster-identity-cred
   config:
+    clusterLabels:
+      k0rdent: demo    
```

Comment:
do we need it? i mean, there is already the scheme and the chart itself, so I believe we already have enough (apart from the docs)

Reply:
@bnallapeta suggested to add it above as an example.

Reply:
we could leave it here but empty, so both cases are covered: we have an example, and no undesired labels applied by default

to sum up

```yaml
spec:
  # ...
  config:
    clusterLabels: {}
  # ...
```

Reply:
@zerospiel seems fair enough-  let's do that. Changes made.